### DISCUSSION TITLE
Docs about variant attributes in svelte are unclear

### DISCUSSION THREAD

**Discussion by Evertt** [windicss/windicss#52] [Q&A] [answered]
In [this part of the docs](https://windicss.netlify.app/guide/svelte.html#additional-features-in-svelte-%E2%9A%A1%EF%B8%8F) you talk about the variant attributes feature, which is awesome, and you write:

>**Mixed Variants**  
>  
>Variants, such as sm:hover, are not supported inside attributes. Use [utility groups](https://windicss.netlify.app/guide/features.html#%F0%9F%8E%B3-utility-groups) instead.

And I'm not sure if I understand what you mean. Do you mean you can't write this?

    <div sm:hover="bg-red-500"></div>

Because if that's what you mean then sure, I get that, because I imagine that svelte would try to interpret those attributes and throw errors. But then I wonder, can you still do this?

    <div sm="hover:bg-red-500"></div>

Because if not, why not? I don't see a reason why that couldn't work.

**Comment by voorjaar** [comment, marked as answer]
`<div sm="hover:bg-red-500"></div>` works;
`<div sm:hover="bg-red-500"></div>` not;

For the above, we can simply merge it into the class attribute, and then hand it over to windicss. The following requires us to parse `sm:hover` again to ensure that it is a usable variant, which requires a lot of extra work and may be very rarely used, so I did not implement this.

**Reply by Evertt** [reply]
Okay cool. Might be good to make that a bit more clear in the docs. :-)

**Reply by voorjaar** [reply]
ok, will do it.
